README.md

```diff
@@ -0,0 +1,304 @@
+```
+
+### 3. (RECOMMENDED) Create a DNS Entry
+
+If you installed the gateway without enabling support for an ingress controller,
+this command should help you find the gateway's public IP address:
+
+```console
+$ kubectl get svc brigade-acr-gateway \
+  --namespace brigade-acr-gateway \
+  --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+If you overrode defaults and enabled support for an ingress controller, you
+probably know what you're doing well enough to track down the correct IP without
+our help. 😉
+
+With this public IP in hand, edit your name servers and add an `A` record
+pointing your domain to the public IP.
+
+### 4. Create Webhooks
+
+In your browser, visit the Azure Portal and navigate to the Azure Container
+Registry for which you'd like to send webhooks to this gateway. From the
+__Services__ section, select __Webhooks__ and click __Add__.
+
+Here, you can add webhooks for the entire registry or for specific repositories
+within the registry.
+
+* In the __Webhook name__ field, add a meaningful name for the webhook.
+
+* If your registry is replicated across regions, select the applicable
+  __Location__.
+
+* In the __Service URI__ field, use a value of the form 
+  `https://<DNS hostname or publicIP>/events`.
+
+* In the __Custom headers__ field, add `Authorization: Bearer <token>`, where
+  `<token>` is any of the tokens that were specified in `my-values.yaml` at the
+  time of gateway installation. This will enable authentication to this gateway.
+
+* In the __Actions__ field, select the actions that should trigger a webhook.
+  Note that `chart_push` and `chart_delete` are not supported by this gateway.
+
+* The __Status__ field can be used to enabled or disable the webhook. It is
+  enabled by default.
+
+* The __Scope__ field can be used to make this webhook be triggered only by the
+  selected action (__Actions__ field) on a _specific repository within the
+  registry_. If __Scope__ is left blank, selected actions on _any repository
+  within the registry_ will trigger the webhook.
+
+* Click __Create__
+
+### 5. Add a Brigade Project
+
+You can create any number of Brigade projects (or modify an existing one) to
+listen for events that were sent from an ACR repository to your gateway and, in
+turn, emitted into Brigade's event bus. You can subscribe to all event types
+emitted by the gateway, or just specific ones.
+
+In the example project definition below, we subscribe to all events emitted by
+the gateway, provided they've originated from the `example.azurecr.io` registry
+and the `brigadecore/brigade-acr-gateway` repository (see the `registry`
+qualifier and `repo` label).
+
+```yaml
+apiVersion: brigade.sh/v2-beta
+kind: Project
+metadata:
+  id: acr-demo
+description: A project that demonstrates integration with ACR
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/acr
+    types:
+    - *
+    qualifiers:
+      registry: example.azurecr.io
+    labels:
+      repo: brigade-acr-gateway
```

Comment:
To match the description above, should the full `brigadecore/brigade-acr-gateway` string be used here?  If so, the same applies to the example below and the one in the `examples/` dir.

Reply:
Nice catch.

It's the description that's wrong. User/org namespaces like those we're familiar with on Docker Hub, quay.io, etc. are not common in ACR.

I'll make the description match the example.

But...

You're also making me second guess something. Across several gateways, I've used the gateway's own repo (or an imaginary repo containing the gateway's name) as examples. This practice possibly could introduce some confusion by making the examples less clear.

I think I should revise all docs/examples to use fictitious repos and explicitly call that out as well.